src/components/Card.js

```diff
@@ -64,19 +65,33 @@ const flipCard = (event, value) => {
 };
 
 const Card = (props) => {
-  const { index } = props;
+  const { index, flipCardClickHandler, cardState } = props;
+  // const [isFlipped, setIsFlipped] = useState(false);
+  const [image, setImage] = useState(constants.blueBGurl);
+  // const id = constants.cardArray[index].name;
+
+  // const [dataId, setDataId] = useState(id);
+
+  // const flippedImageUrl = constants.cardArray[index].img;
 
   return (
     <ScoreConsumer>
       {(value) => {
         return (
           <img
-            data-id={constants.cardArray[index].name}
-            src={constants.blueBGurl}
+            src={image}
             className="card-container"
             alt=""
-            data-index={index}
-            onClick={(event) => flipCard(event, value)}
+            index={index}
+            cardState={cardState}
+            onClick={
+              flipCardClickHandler
```

Comment:
this is a method, and `onClick` should look like the method you deleted. So:

```js
onClick={(event) => flipCardClickHandler(event, index)}
```

Although `index` shouldn't be passed in since you don't use that parameter even though you specified it takes an `index` parameter on L40 of `src/components/Board.js`